src/test/java/org/salesforce/steps/CreateIndividualSteps.java

```diff
@@ -10,6 +10,8 @@
 
 import com.fasterxml.jackson.databind.ObjectMapper;
 import java.util.Map;
+

```

Comment:
This might cause Checkstyle errors, remove the space and order the imports lexicopgraphically.